README.md

```diff
@@ -1 +1,54 @@
-# restaurant-page
\ No newline at end of file
+## restaurant-page
+ This project comes after the introduction of the ES6 class keyword, ES6 modules, and webpack. The main goal of this project is for the student to show the understanding of the benefits of modularized code as well as the ability to set up a javascript application made of different modules using webpack.
+
+
+ ## Screen shot of Page
+<img src="./src/assets/restaurant-page.png">
+
+
+ ## Page live Demo Link
+
+ [link!](https://raw.githack.com/dasileker/restaurant-page/feature/dist/index.html)
+
+ ## Built With
+
+ - Only Javascript 
+ - Webpack
+ - Used the DOM Manipulation.
+ - CSS
+ - Images
+ - Linters
+
+ ## How It work 
+  - Frist Open Your terminal and paste : git clone git@github com:dasileker/restaurant-page.git.
+
+  - Thene cd to the Folder && Open it in your Texteditor.
+
+  - To Make the website fully working make sure to install the webpack
+  and run the folowing command if yoy don't have it.
+
+  - npm install webpack -g "to make the installation global".
+
+  - npm install webpack-cli --save-dev.
+  - add this comand if you want to work with the webpack server " npm install --save-dev webpack-dev-server.
+
+  - voila Now youcan view the page in your localhost ENV.
```

Comment:
## Improve  the README
- [ ] I feel these instructions may not be necessary. I would suggest we remove them and only add the features of running the webapck in the `package.json` file using the `start` script. What do you think?